tools/cldr-apps/src/main/java/org/unicode/cldr/web/VettingViewerQueue.java

```diff
@@ -305,7 +305,7 @@ private HashMap<String, String> getLocaleToProgress(VettingViewer<Organization>
             final ArrayList<String> localeList = vv.getLocaleList(org);
             for (String localeId : localeList) {
                 final CLDRLocale loc = CLDRLocale.getInstance(localeId);
-                final LocaleCompletion.LocaleCompletionResponse response = LocaleCompletion.handleGetLocaleCompletion(loc);
+                final LocaleCompletion.LocaleCompletionResponse response = LocaleCompletion.getLocaleCompletion(loc);
```

Comment:
@btangmu this was calling the uncached worker function before. Perhaps because the permissions were backwards.